src/views/createprojectmodal/constants.ts

```diff
@@ -0,0 +1,67 @@
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+import { t } from '@utils/hooks/useNetworkingTranslation';
+
+type UDNSpec = {
+  layer2: {
+    ipamLifecycle: string;
+    role: string;
+    subnets: string[];
+  };
+  topology: string;
+};
+
+export const DESCRIPTION_ANNOTATION = 'openshift.io/description';
+export const DISPLAY_NAME_ANNOTATION = 'openshift.io/display-name';
+
+export enum NETWORK_TYPE {
+  POD_NETWORK,
+  UDN,
+  // CLUSTER_UDN,
+}
+
+export const networkTypeLabels = {
+  [NETWORK_TYPE.POD_NETWORK]: t('Use the default Pod network'),
+  [NETWORK_TYPE.UDN]: t('Define a new UserDefinedNetwork for this project'),
+  // eslint-disable-next-line perfectionist/sort-objects
+  // [NETWORK_TYPE.CLUSTER_UDN]: t('Refer an existing ClusterUserDefinedNetwork'),
```

Comment:
??

Reply:
@upalatucci

Reply:
We'll un comment this code when the Cluster UDN will be ready and available in the cluster